app/views/locations/edit.html.erb

```diff
@@ -44,6 +44,10 @@
                                            hint: { text: "This includes stairlifts, ramps, level-access showers or grab rails" },
                                            legend: { text: "Are the majority of units in this location built or adapted to wheelchair-user standards?", size: "m" } %>
 
+      <%= f.govuk_date_field :startdate,
+                              legend: { text: "When did the first property in this location become available?", size: "m" },
```

Comment:
@kosiakkatrina worth using the translation file for this since we have the string in multiple places?